CIP-?/README.md

```diff
@@ -0,0 +1,99 @@
+
+## Specification
+
+1. **Naming Conflict Detection:**
+
+   - When a user enters a domain name (e.g., `hinson.ada`) with the same prefix or suffix, the wallet should detect the potential conflict. Possibly by attempt to resolve all potential project's corresponding addresses.
+   - When a user enters a domain name (e.g., `hinson.ada`) with the same prefix or suffix, the wallet should detect the potential conflict by attempting to resolve all potential project's corresponding addresses.
+
+2. **User Prompt:**
+
+   - If there is only one corresponding address resolved, the wallet could proceed with the only address for proceeding with the user transaction.
+   - If more than one corresponding address is detected, the wallet should display a prompt to the user, explaining the conflict, and providing options for resolution.
+
+3. **Resolution Options:**
+
+   - The prompt should offer the user the following options:
+
+     - Resolve with Project A
+     - Resolve with Project B
+     - etc...
+     - Cancel
+
+     > Detail UX implementation to be decided by the wallet as long as users are not confused with their choice, so only showing the project name is the minimal information needed, e.g. one for Cardano Name Service and another for adadomain
+
+4. **User Selection:**
+
+   - The user selects one of the options presented in the prompt. If they choose to resolve with Project A, the Project A's resolver is used; if they choose the other project, the respective resolver is used.
+
+5. **Default Project:**
+
+   - Users may have the option to set a default project for names with a colliding suffix in their wallet settings. This default project will be used for resolution unless changed by the user during the resolution prompt.
+   - Without user specification, there should always be an option prompting to user in case of domain name conflict.
+
+## Rationale: how does this CIP achieve its goals?
+
+This proposal embraces the decentralization property of a blockchain where it welcomes multiple domain name service that exists in the market without user experience competition at the infrastructure level. It ensures a user-friendly experience and allows users to make informed decisions in case of naming conflicts. It accommodates multiple projects with the same suffix in wallets while avoiding disputes and confusion, fostering the integration process of wallet and domain service.
+
+### Path to Active
+
+#### Acceptance Criteria
+
+- At least 3 of wallets listed below agree with the approach
+
+  - Begin <https://begin.is/>
```

Comment:
```suggestion
  - [ ] Begin <https://begin.is/>
```
It would help everyone (including your potential implementors) to have these items also as tick-boxes for when you get buy-in from the wallets (with a parent tick-box when you've got >= 3 of them), if you continue to follow this criterion.